packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -322,6 +327,93 @@ export function nft_revoke_all(token_id: string): void {
     }
 }
 
+/* Storage management */
+
+function internal_storage_balance_of(
+    account_id: string
+): StorageBalance | null {
+    if (accounts.get(account_id)) {
+        return <StorageBalance>{
+            total: storage_balance_bounds().min,
+            available: '0',
+        }
+    } else {
+        return null
+    }
+}
+
+export function storage_deposit(
+    account_id: string | null,
```

Comment:
I'm surprised this works considering [`AssemblyScript` does not support Union Types](https://www.assemblyscript.org/basics.html#no-union-types).